os/arch/arm/src/common/up_internal.h

```diff
@@ -278,7 +278,7 @@ EXTERN uint32_t _edata;			/* End+1 of .data */
 EXTERN uint32_t _sbss;			/* Start of .bss */
 EXTERN uint32_t _ebss;			/* End+1 of .bss */
 EXTERN uint32_t _sidle_stack;		/* Start of idle stack */
-EXTERN uint32_t __sint_heap_start;	/* Start of heap in internal RAM region */
+EXTERN uint32_t _sint_heap_start;	/* Start of heap in internal RAM region */
 EXTERN uint32_t __sext_heap_start;	/* Start of heap in external RAM region */
```

Comment:
@thapav , 
in the previous commit (https://github.com/Samsung/TizenRT/commit/b36bd18a57093fbaa6c23aa0d3c26621c119d5e6) where _sint_heap_start and _sext_heap_start were added, in up_internal.h they were added with an extra underscore. Could you please confirm if its requried?

Reply:
> @thapav , in the previous commit ([b36bd18](https://github.com/Samsung/TizenRT/commit/b36bd18a57093fbaa6c23aa0d3c26621c119d5e6)) where _sint_heap_start and _sext_heap_start were added, in up_internal.h they were added with an extra underscore. Could you please confirm if its requried?

It must be '__s**_heap_start'
Please refer to https://github.com/Samsung/TizenRT/blob/master/docs/HowToAddnewBoard.md

Reply:
@thapav The prefix `_s` means `start`, `_e` does `end`. Then can we remove post fix `_start`?
I think `_s**_heap_start` has two `start`s. Could you check and make a new PR for this if necessary?